transforms/htmlTemplateCleanup/index.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { fixHtmlTemplateDirectives } from './invalidTemplateAttributes.js'
+
+const transforms = [fixHtmlTemplateDirectives]
+
+export const htmlTemplateCleanup = async ({ templates }) => {
+  const result = {
+    overwrite: {},
+    delete: []
+  }
+
+  templates.forEach(template => {
+    transforms.forEach(transform => transform(template, result))
+  })
+
```

Comment:
@nolanlawson I plan to combine the `parse5` mods into one mod but keep them separated into two separate files.

I plan to combine them into a single mod here. 

I set it up this way in case we decide to add more mods into this codemod which should serve as a general codemod to fix any HTML warnings/errors.